tests/suites/test_suite_ssl.function

```diff
@@ -158,8 +159,6 @@ int mbedtls_test_buffer_get( mbedtls_test_buffer *buf,
  * Errors used in the message transport mock tests
  */
  #define MBEDTLS_TEST_ERROR_ARG_NULL -11
- #define MBEDTLS_TEST_ERROR_QUEUE_FULL -22
- #define MBEDTLS_TEST_ERROR_QUEUE_EMPTY -33
  #define MBEDTLS_TEST_ERROR_MESSAGE_TRUNCATED -44
 
```

Comment:
I think ERROR_ARG_NULL could be also removed here and replaced in the code, by SSL_BAD_INPUT_DATA. SSL_BAD_INPUT_DATA is more often used in case of NULL pointers, so I think using this value would more in line with the convention